Dockerfile

```diff
@@ -1,30 +1,44 @@
 FROM python:3.6
+
+ARG USER_ID
+ARG GROUP_ID
+ENV USER='test'
+ENV DEF_USER_ID=1000
+ENV DEF_GROUP_ID=1000
+
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt-get -qq update; apt-get -y install nodejs gettext libgettextpo-dev libgraphviz-dev libz-dev libjpeg-dev libfreetype6-dev python3-dev gdal-bin python-numpy locales locales-all
-ENV LC_ALL en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US.UTF-8
```

Comment:
Proč jsi tohle odstranil? Je to důležité, spoustů věci nefunguje bez nastavení LANGu, je to zakladní obraz je prostě rozbité bez tyhle řádky.

Reply:
Čo konkrétne je na tom závislé? (asi som prehliadol, keď som to testoval) Mne zostavenie image/containeru prebehlo v poriadku, aj bez týchto riadkov.

Reply:
no, když debuguješ, tak nefunguje v podstatně žádné utils jako je vim nebo top, a taky dochazí k náhodným chybkám ve neočekavaných mistech. Tyhle řádky by bohužel měli být ve každém Dockerfile založení na obrazu debian, je to prostě zbytečné útrpění pokud je tam nemáš.

Reply:
V poriadku, vďaka za vysvetlenie, doplním to.